src/app/upload/view.tsx

```diff
@@ -132,28 +139,21 @@ export const UploadView: NextPage<PropsUploadPage> = ({ session }) => {
           Your Tracks
         </h3>
         <div className="scrollbar flex h-2/4 flex-col gap-5 overflow-y-scroll py-6">
-          {isFetched &&
-            loadedTracks!.map((item: Track, index: number) => (
-              <TrackItem
-                currentTrackId={currentTrack?.id}
-                key={item.id}
-                track={item}
-                deleteFile={deleteFile}
-                updateFile={updateFile}
-                handleTrack={setCurrentTrack}
-                index={index + 1}
-              />
-            ))}
+          {isFetched && loadedTracks?.length
+            ? loadedTracks.map((item: Track, index: number) => (
```

Comment:
Usually, when mapping an array you can omit the type of method callback parameters.
```suggestion
            ? loadedTracks.map((item, index) => (
```